test/operators.jl

```diff
@@ -58,6 +58,25 @@ p = 1=>:foo
 @test_throws ArgumentError Base.scalarmax('a',['c','d'])
 @test_throws ArgumentError Base.scalarmax(['a','b'],'c')
 
+s1 = Set([1])
+s2 = Set([2])
+@test_throws MethodError min(s1, s2)
+@test_throws MethodError max(s1, s2)
+@test_throws MethodError minmax(s1, s2)
+
+struct TO
```

Comment:
Perhaps use a more specific type name, like `T23094` or something for the issue number? Also, 4 space indent for the `x`.